src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunTest.java

```diff
@@ -473,4 +479,17 @@ private void assertCulprits(WorkflowRun b, String... expectedIds) throws IOExcep
         assertFalse(b.isLogUpdated());
     }
 
+    @Test public void getScms() throws Exception {
+        sampleRepo.init();
+        sampleRepo.write("Jenkinsfile", "echo 'hello'");
+        sampleRepo.git("add", "Jenkinsfile");
+        sampleRepo.git("commit", "--message=files");
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        CpsScmFlowDefinition def = new CpsScmFlowDefinition(new GitStep(sampleRepo.toString()).createSCM(), "Jenkinsfile");
```

Comment:
nit: No need to go through `GitStep` like this, I think you can just create a `GitSCM` directly:

```suggestion
        CpsScmFlowDefinition def = new CpsScmFlowDefinition(new GitSCM(sampleRepo.toString()), "Jenkinsfile");
```

(and then remove the GitStep import)